src/convert/streams.ts

```diff
@@ -54,6 +53,7 @@ export class ComponentReader extends Readable {
     this.push(null);
   }
 
+  // eslint-disable-next-line class-methods-use-this
```

Comment:
this is just to avoid making this static correct?

Reply:
I left it.  There's a good case to be made for getting rid of componentReader altogether (see the note in the handbook) now that there's ReadableStream from an array.

But that's felt like a separate PR. 

This was more of a "try to leave it"